filesystem/dirs_test.go

```diff
@@ -136,6 +137,31 @@ func TestGetSpacemeshTempDirectoryPath(t *testing.T) {
 	TearDownTestHooks()
 }
 
+func TestEnsureDataSubDirectory(t *testing.T) {
+	users := TestUsers()
+	SetupTestHooks(users)
+	usr, err := currentUser()
+	assert.NoErr(t, err, "getting current user failed")
+
+	testCases := []struct {
+		user     *user.User
+		expected string
+		error    bool
+	}{
+		{usr, "~" + RootFolder + ".spacemesh" + RootFolder + "temp", false},
```

Comment:
Since we ain't using `RootFolder` as the actual root folder, technically what you did works but 
its a little bit misleading to use it here, either rename it, or use an actual backslash.